bot.py

```diff
@@ -42,6 +51,10 @@ def __init__(self, *args, **kwargs):
 async def on_command_error(ctx, error):
     if isinstance(error, CommandNotFound):
         return
+
+    # send error to sentry.io
+    bot.log_exception(error)
```

Comment:
You'll probably want to ignore `ReportException`s (most commonly used as "report not found")

Reply:
Ok, added